ros2/manifest.yaml

```diff
@@ -47,19 +52,52 @@ release_names:
                         <<: *DEFAULT
                         archs:
                             - amd64
+                            - arm64v8
                         tag_names:
                             ros-core:
                                 aliases:
                                     - "$release_name-ros-core"
+                                    - "$release_name-ros-core-$os_code_name"
                             ros-base:
                                 aliases:
                                     - "$release_name-ros-base"
+                                    - "$release_name-ros-base-$os_code_name"
                             desktop:
                                 aliases:
                                     - "$release_name-desktop"
+                                    - "$release_name-desktop-$os_code_name"
                             ros1-bridge:
                                 aliases:
                                     - "$release_name-ros1-bridge"
+                                    - "$release_name-ros1-bridge-$os_code_name"
+    crystal:
+        eol: 2020-XX
+        os_names:
+            ubuntu:
+                os_code_names:
+                    bionic:
+                        rosdistro_name: melodic
+                        <<: *DEFAULT
+                        archs:
```

Comment:
Would be nice to target arm32v7 as well, but I'm guessing that's not on the buildfarm roadmap?

Reply:
@nuclearsandwich says `It's Complicated [...] at release the only planned architectures for crystal are arm64 and amd64`

Reply:
... I can dream. @nuclearsandwich , if there is anything I could do to help in advancing arm32v7, like debug tests or investigate issues, I gladly give it go. There are a lot of hobbyist and academic users that use armhf, and if we had readily available images they could easily pull from, I think it would really help bolster ROS2 exposure and adoption.

Reply:
@ruffsl I don't want to sidetrack this discussion too much, there's a lot that I think can be done. For one, I'm not sure if anyone has done a from-source build of ROS 2 on an armv7 system recently. If it's something you're interested in I'd suggest a discourse thread.